gsy_framework/utils.py

```diff
@@ -507,7 +507,8 @@ class HomeRepresentationUtils:
 
     @staticmethod
     def _is_home(representation):
-        home_devices = ["PV", "Storage", "Load", "MarketMaker"]
+        home_devices = ["PV", "Storage", "Load", "MarketMaker", "HeatPump", "WindTurbine",
```

Comment:
I am tempted to suggest adding `VirtualHeatpump` to the list for future compatibility, but ignoring it due to YAGNI.